bean/internal/usecase/passwordless/passwordless.go

```diff
@@ -95,19 +97,25 @@ func (u *UseCase) findOrCreateUser(email string) (*model.User, error) {
 	return user, nil
 }
 
-func buildEmailBody(tokenID, password string) string {
+func (u *UseCase) buildEmailBody(tokenID, password string) string {
```

Comment:
For clarification, I chose to keep this method in the use case for simplicity. 

It's the only case we have of generating email content. No need to over engineer.